src/vicsetup/Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: tests
```

Comment:
Why does this need its own Makefile? Can't this be build recursively with the Makefile under `src/`?

Reply:
This makefile is not used when building libsgxlkl.so. Instead we add rules to src/Makefile as you suggest.

Reply:
This makefile is for building the libvicsetup, the vicsetup test tool, and the tests.